client/src/handlers/handlers.ts

```diff
@@ -311,9 +174,9 @@ function runReplCmd<A extends unknown[]>(
     await Promise.allSettled([handleUnsavedFiles(), prepareRepl()])
 
     if (typeof cmd === 'string') {
-      FLIX_TERMINAL.sendText(cmd)
+      flixTerminal.sendText(cmd)
     } else {
-      FLIX_TERMINAL.sendText(await cmd(...args))
+      flixTerminal.sendText(await cmd(...args))
```

Comment:
Does this work correctly? E.g. clicking run tests etc?

Reply:
Yep, it appears to